teamBadges.json

```diff
@@ -0,0 +1,154 @@
+window.teamBadges = {
```

Comment:
created a global reference within the JSON file to make sure it is available when imported in the userscript